packages/openapi-to-graphql/src/resolver_builder.ts

```diff
@@ -40,13 +49,272 @@ type GetResolverParams = {
+ */
+export function getSubscribe({
+  operation,
+  argsFromLink = {},
+  payloadName,
+  data,
+  baseUrl,
+  connectOptions
+}: GetSubscribeParams): SubscriptionIterator {
+  // Determine the appropriate URL:
+  if (typeof baseUrl === 'undefined') {
+    baseUrl = Oas3Tools.getBaseUrl(operation)
+  }
+
+  // Return custom resolver if it is defined
+  const customResolvers = data.options.customResolvers
+  const title = operation.oas.info.title
+  const path = operation.path
+  const method = operation.method
+
+  if (
+    typeof customResolvers === 'object' &&
+    typeof customResolvers[title] === 'object' &&
+    typeof customResolvers[title][path] === 'object' &&
+    typeof customResolvers[title][path][method] === 'object' &&
+    customResolvers[title][path][method].hasOwnProperty('subscribe')
+  ) {
+    translationLog(
+      `Use custom subscribe resolver for ${operation.operationString}`
+    )
+    const customResolver = customResolvers[title][path][method] as ResolveObject
+    if (typeof customResolver.subscribe === 'function') {
+      return customResolver.subscribe
+    }
+  }
+
+  return (root: any, args, ctx, info = {}) => {
+    /**
+     * Determine possible topic(s) by resolving callback path
+     *
+     * GraphQL produces sanitized payload names, so we have to sanitize before
+     * lookup here
+     */
+
+    const paramName = Oas3Tools.sanitize(
+      payloadName,
+      Oas3Tools.CaseStyle.camelCase
+    )
+
+    let resolveData: any = {}
+
+    if (
+      root &&
+      typeof root === 'object' &&
+      typeof root['_openAPIToGraphQL'] === 'object' &&
+      typeof root['_openAPIToGraphQL'].data === 'object'
+    ) {
+      const parentIdentifier = getParentIdentifier(info)
+      if (
+        !(parentIdentifier.length === 0) &&
+        parentIdentifier in root['_openAPIToGraphQL'].data
+      ) {
+        /**
+         * Resolving link params may change the usedParams, but these changes
+         * should not be present in the parent _openAPIToGraphQL, therefore copy
+         * the object
+         */
+        resolveData = JSON.parse(
+          JSON.stringify(root['_openAPIToGraphQL'].data[parentIdentifier])
+        )
+      }
+    }
+
+    if (payloadName && typeof payloadName === 'string') {
+      // The option genericPayloadArgName will change the payload name to "requestBody"
+      const sanePayloadName = data.options.genericPayloadArgName
+        ? 'requestBody'
+        : Oas3Tools.sanitize(payloadName, Oas3Tools.CaseStyle.camelCase)
+
+      if (sanePayloadName in args) {
+        if (typeof args[sanePayloadName] === 'object') {
+          const rawPayload = Oas3Tools.desanitizeObjKeys(
+            args[sanePayloadName],
+            data.saneMap
+          )
+          resolveData.usedPayload = rawPayload
+        } else {
+          const rawPayload = JSON.parse(args[sanePayloadName])
+          resolveData.usedPayload = rawPayload
+        }
+      }
+    }
+
```

Comment:
Resolve request body of the callback to evaluate the path ( runtime expression ) that will be used the topic of the subscription.